### PR TITLE
docs: Fix typo

### PR DESCRIPTION
**Pull request by vatsashah45**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?

Step-by-steps

## What is the new behavior?

Step-by-step

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/guides/localization.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This guide will walk you through the necessary steps to localize an Uno Platform
 > [!TIP]
 > For a step-by-step guide to installing the prerequisites for your preferred IDE and environment, consult the [Get Started guide](../get-started.md).
 
-## Step-by-steps
+## Step-by-step
```
Comment (Youssef1313) [inline doc/articles/guides/localization.md:15]:
```suggestion
## Step-by-step instructions
```

File: doc/articles/splash-screen.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ While the new templates simplify adding a splash screen, this article covers how
 >
 > See the guide [How-To: Get Started with Uno.Resizetizer](xref:Uno.Resizetizer.GettingStarted#unosplashscreen) for more information.
 
-## Step-by-steps
+## Step-by-step
```
Comment (Youssef1313) [inline doc/articles/splash-screen.md:16]:
```suggestion
## Step-by-step instructions
```

**Review by Youssef1313 — APPROVED** [review]

**Comment by Youssef1313** [comment]
@Mergifyio backport release/stable/5.3

**Review by kazo0 — APPROVED** [review]

**Pull request merged**